src/filter/delay_filter.rs

```diff
@@ -80,11 +80,8 @@ impl filter::Filter for DelayFilter {
             metric::Event::TimerFlush(f) => {
                 res.push(metric::Event::TimerFlush(f));
             }
-            metric::Event::Raw { encoding, bytes } => {
-                res.push(metric::Event::Raw {
-                    encoding: encoding,
-                    bytes: bytes,
-                });
+            raw @ metric::Event::Raw {..} => {
```

Comment:
Oh hey, this is new syntax to me. What's this?

Reply:
Binds `raw` as an identifier and doesn't care about the contents of the enum struct.

Reply:
Well hell, that'll clean up other bits of cernan.